### PR TITLE
Don't include shippingAddress in mutation input

### PR DESCRIPTION
**Pull request by barryWarnock**
issue: https://github.com/Shopify/shop-pay-for-ccs-team/issues/632

### CODE REVIEW

**Review by ThisIsOstad — APPROVED** [review]
LGTM

File: cartridges/int_shoppay/cartridge/scripts/shoppay/storefrontAPI.js [hunk 1]
```diff
@@ -10,8 +10,9 @@ var Result = require('dw/svc/Result');
  * @returns {Object} The GraphQL service response body
  */
 function shoppayPaymentRequestSessionCreate(basket, paymentRequest) {
-    if (paymentRequest.shippingAddress && paymentRequest.shippingAddress.id) {
-        delete paymentRequest.shippingAddress.id;
+    if (paymentRequest.shippingAddress) {
+        // shipping address was depreciated in ShopPayPaymentRequestInput as of 2024-07
```
Comment (jamiely) [inline cartridges/int_shoppay/cartridge/scripts/shoppay/storefrontAPI.js:14]: deprecated

**Review by jamiely — COMMENTED** [review]
Where do you change the API version from `unstable` to `2024-07`? Is that a separate PR or are we not going to do that?

File: cartridges/int_shoppay/cartridge/scripts/shoppay/storefrontAPI.js (hunk 1 above)
Comment (barryWarnock) [reply cartridges/int_shoppay/cartridge/scripts/shoppay/storefrontAPI.js:14]: damnit, again. Thanks Jamie

**Review by jamiely — APPROVED** [review]

**Pull request merged**